src/main/java/uk/gov/hmcts/reform/cpo/validators/Validator.java

```diff
@@ -1,16 +1,36 @@
 package uk.gov.hmcts.reform.cpo.validators;
 
+
 import javax.validation.ConstraintValidatorContext;
 import java.util.List;
+import java.util.UUID;
+
+import static org.apache.commons.validator.routines.checkdigit.LuhnCheckDigit.LUHN_CHECK_DIGIT;
 
 interface Validator<T> {
 
+    String CASE_ID_RG = "\\d{16}";
 
     default void buildErrors(final ConstraintValidatorContext context, final String message, List<String> errors) {
         final String errorsCommaSeparated = String.join(",", errors);
         context.buildConstraintViolationWithTemplate(message + errorsCommaSeparated + " are incorrect.")
             .addConstraintViolation();
+    }
+
+    static boolean isValidCaseId(String caseId) {
+
```

Comment:
I would rather not make it static. I would avoid using statics.

Reply:
Resolved in 94d7cc77493c1245220ac584184bb0099a9c3904